translations/wesnoth-A_New_Order/es.po

```diff
@@ -1404,42 +1524,50 @@ msgstr ""
 #. [event]
 #: A_New_Order/macros/deaths.cfg:43 A_New_Order/scenarios/23_Trapped.cfg:584
 #: A_New_Order/scenarios/23_Trapped.cfg:597
+#, fuzzy
 msgid "Dunefolk die without fear!"
-msgstr ""
+msgstr "¿Muero sin conseguir lah josha?"
```

Comment:
So, I'm realizing, this string is used in 3 different contexts... I'm wondering if I should use the caret method of differentiating the strings so that the line can have different translations depending on the line that it's followed up with?

Reply:
That fuzzy was added automatically added by poedit believing that there are similarities with a translation of Legends of Wesmere, I think xD

Reply:
ok, but regardless of the translation, should I make the modification to the source string(s) that I was asking about?

Reply:
Ah, i'll look at it later

Reply:
I have seen all three contexts and would give the same translation to all three

Reply:
ok, resolving...